packages/search-field/src/SearchField.js

```diff
@@ -212,8 +213,17 @@ class SearchField extends React.Component {
     const { focused, selectedItem } = this.state;
 
     const Input = selectedItem ? ReadOnly : EnhancedInput;
+
     Input.displayName = "Input";
 
+    // Focus the input component after
+    // selecting an item and then clearing it
+    // by clicking on the pencil icon
+    const autoFocus =
```

Comment:
I think this comment is still a bit confusing. My suggestion:
```javascript
// When there's an item selected and we click the edit icon,
// the input ref points to the StaticInput instance, which has no focus().
// By setting the component focused prop, the regular
// input will get focus when it gets mounted.
```

Reply:
Thanks!